### DISCUSSION TITLE
Raspberry Pi 5: Power supply

### DISCUSSION THREAD

**Discussion by outdoorbits** [outdoorbits/little-backup-box#272] [General]
The [case](https://github.com/outdoorbits/case-for-little-backup-box/tree/main/Raspberry_Pi_5) for Raspberry Pi 5 is finished and works well. Buttons and display had problems because of the new hardware chip RP1 but these problems are solved now. Even comitup is working using the new os version bookworm.
The Pi 5 is significantly faster in local file transfer and generating preview images.
Together with the original power supply you can connect your USB drives directly to the Pi - a powered USB hub is not needed any more.
But: For travelling the new power supply is (too) heavy, (too) big and the cable is permanently connected. I hoped I could manage power supply by a Y-power cable, using two ports of my travel USB power supply, but this didn't work in first tests.
Actually I'm using my travel power supply together with my powered USB hub. I do not understand why, but the works well.
Does anybody have another clever and slim solution?

**Comment by outdoorbits** [comment]
Good news, latest version of lbb sets usb_max_current_enable=1. This prevents raspberry pi 5 to reduce usb max current in case the pi isn't connected to a 5V/5A power supply. It looks like the y-power-cable does it's job now!

**Reply by thelightdecides** [reply]
Thats really good news!

**Comment by GypsyAcidQueen** [comment]
Hey @outdoorbits sorry noob here! Does this y power cable mean we can use a PD-battery to power the pi and attached devices?

**Comment by outdoorbits** [comment]
My pi 5 is usually powered by y-cable. If your power bank has two 
outputs of 5 V and at least 2.5 A (12.5 W) or better each, this should 
work. USB-c outputs can deliver higher voltage and higher power on 
suitable devices - but your pi isn't suitable for that. It needs 5V. 
USB-A outputs are always 5V (in my opinion). Pi 4 usually needs a 
powered usb hub.

**Reply by GypsyAcidQueen** [reply]
Got it, thanks. Which specific power bank and powered USB hub are you using, just out of interest? I might try and pick one up just because it's confirmed as working. 

I've been looking into other solutions but portable 5v/5a just seems like an absolute nightmare to get working. There are some potential solutions in this thread: https://forums.raspberrypi.com/viewtopic.php?t=361748 but nothing that's sleek enough to make me want to try it

**Comment by outdoorbits** [comment]
My USB hub is this one:

https://www.orico.shop/de/usb-30-hub-mit-4-usb-a-anschluessen-zusaetzliches.html

I couldn't find it anywhere but I think virtually every powered USB 3.0 
hub will work.

For the power bank my device is similar to Anker 337 Powerbank. It has 
three USB-A outputs, two for the Pi and one for the USB hub if 
necessary. All together it delivers up to 6A x 5V =30 W.
